app/actions/handlers.py

```diff
@@ -276,21 +276,25 @@ async def process_attachments(transformed_data, response, integration):
                     attachments=[(filename, img)],
                     integration_id=integration.id
                 )
-        except Exception as e:
+        except httpx.HTTPStatusError as e:
+            if e.response.status_code == 403:
+                message = f"HTTP 403 Forbidden response while reading event attachment for event ID '{event_id['object_id']}'. Exception: {e}"
+            else:
+                message = f"Error while processing event attachment for event ID '{event_id['object_id']}'. Exception: {e}"
+
```

Comment:
@vgarcia13 Do you know why this is happening? Do you think the body of the response might provide more details?

My hunch is that the token is invalid and that we need to retry this request after getting a new token.